src/pykx/pandas_api/pandas_meta.py

```diff
@@ -255,3 +255,12 @@ def sum(self, axis=0, skipna=True, numeric_only=False, min_count=0):
             res,
             min_count
         ), cols)
+    
+    @convert_result
+    def nunique(self, axis=0, dropna=True):
+        res, cols = preparse_computations(self, axis, skipna=False)
+        filterNan = q('{$[11h = type x;x;'
```

Comment:
They only use this function for dropna:
```q
x where not null x
```
Your function is more complete and if it works I would even make it an auxiliary function so that it can be used in other functions, but if not I would not get too involved in this parameter either.

Reply:
Yes, I think it would be good to have something like this in a auxiliary function. I'll note that down as a task